app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -577,13 +577,28 @@ def create_volumes(vm, attrs)
       # add volumes
       volumes = nested_attributes_for :volumes, attrs
       volumes.map do |vol|
-        set_preallocated_attributes!(vol, vol[:preallocate])
-        # The blocking true is a work-around for ovirt bug fixed in ovirt version 3.1.
-        vm.add_volume({:bootable => 'false', :quota => ovirt_quota, :blocking => api_version.to_f < 3.1}.merge(vol)) if vol[:id].blank?
+        if vol[:id].blank?
```

Comment:
We can use `next if vol[:id]` if we switch to `each` earlier, it would be a bit more readable.